docs/blog/20240624-Daeraxa-3kStars.md

```diff
@@ -0,0 +1,136 @@
+
+<img src="./assets/early-atom.png" height="400" />
+
+Of course, the thing that made Atom, Atom, was its underlying framework, something developed specifically for Atom called "Atom Shell", a cross-platform framework designed to allow web technologies to be used in a desktop application. You might know it better these days as [Electron](https://www.electronjs.org/blog/electron/). So yes, if you weren't aware before, this means that the Atom editor is directly responsible for the framework that allows a whole host of popular applications to exist: Discord, VSCode, Joplin, Slack, and [far, far, more](https://www.electronjs.org/apps).
+
+Later in 2014, [GitHub announced](https://web.archive.org/web/20140506163807/http://blog.atom.io/2014/05/06/atom-is-now-open-source.html) that Atom would become fully open-sourced under the MIT license and invited the community to actively contribute to its development; it is under this license that Pulsar is still developed.
+
+In 2015, [Atom 1.0](https://github.com/atom/atom/releases/tag/v1.0.0) was released and came with a [fantastic retro-futurisitic launch video](https://www.youtube.com/watch?v=Y7aEiVwBAdk) to accompany it.
+
+<iframe width="560" height="315" src="https://www.youtube.com/watch?v=Y7aEiVwBAdk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Microsoft acquires GitHub
+
+<img src="./assets/microsoft-github.png" height="200" />
+
+In June of 2018, Microsoft announced that they were [acquiring GitHub](https://blogs.microsoft.com/blog/2018/06/04/microsoft-github-empowering-developers/). This immediately raised concerns from the Atom user base because Microsoft had released their own Electron-based text editor back in 2015, Visual Studio Code. The concerns were that Atom would shut down development of Atom in favor of their own product as the two products were competing in very similar areas. To help alleviate the Atom community's worries, Microsoft's newly instated Github CEO, Nat Friedman, said in a [Reddit AMA](https://www.reddit.com/r/AMA/comments/8pc8mf/im_nat_friedman_future_ceo_of_github_ama/e0a2b2e/) that Atom was safe and development would continue on both products.
```

Comment:
```suggestion
In June of 2018, Microsoft announced that they were [acquiring GitHub](https://blogs.microsoft.com/blog/2018/06/04/microsoft-github-empowering-developers/). This immediately prompted concerns from the Atom user base because Microsoft had released their own Electron-based text editor back in 2015: Visual Studio Code. The concern was that Microsoft would shut down development of Atom in favor of their own product as the two products were competing in very similar areas.

To help alleviate the Atom community's worries, GitHub’s new CEO said in a [Reddit AMA](https://www.reddit.com/r/AMA/comments/8pc8mf/im_nat_friedman_future_ceo_of_github_ama/e0a2b2e/?context=1) that Atom was safe and development would continue on both products.
```

Reply:
I was kind of keen to keep the Microsoft link in there, yes he was GitHub's new CEO but he was moved directly from a senior position within Microsoft rather than recruited externally. Seeing as I've named other important people in the story up until this point it seems like we should keep that in?

Reply:
Didn’t mean to strip that link; I had made edits on your first draft and did an impromptu sort of three-way merge.

Reply:
Actually, which thing did you mean? I replied on my phone this morning and assumed that I had omitted a URL somewhere, but all I did was insert a paragraph break. I'm not clear on what you mean by “the Microsoft link” now.

Reply:
Let's make sure to use the right casing on “GitHub,” though.